js/src/experimental/otel/constants.ts

```diff
@@ -45,6 +45,7 @@ export const LANGSMITH_RUN_ID = "langsmith.span.id";
 export const LANGSMITH_TRACE_ID = "langsmith.trace.id";
 export const LANGSMITH_DOTTED_ORDER = "langsmith.span.dotted_order";
 export const LANGSMITH_PARENT_RUN_ID = "langsmith.span.parent_id";
+export const LANGSMITH_USAGE_METADATA = "langsmith.usage_metadata";
```

Comment:
@angus-langchain seems we don' set this in the python side - any reason for that?

Reply:
Because it's brand new :)